bentoml/_internal/runner/container.py

```diff
@@ -128,6 +129,75 @@ def payload_to_single(
     payload_to_batch = payload_to_single
 
 
+class PandasDataFrameContainer(
+    DataContainer[t.Union["pd.DataFrame", "pd.Series"], "pd.DataFrame"]
+):
+    @classmethod
+    def singles_to_batch(
+        cls, singles: t.Sequence[t.Union["pd.DataFrame", "pd.Series"]], batch_axis=0
+    ) -> "pd.DataFrame":
+        import pandas as pd
+
+        assert batch_axis == 0, "PandasDataFrameContainer requires batch_axis = 0"
+
+        # here we assume each member of singles has the same type/shape
```

Comment:
should we assert this?

Reply:
would it raise an error when we try to merge them into one data frame?  I think either way is ok

Reply:
marked, I will check it later. Merge this pr now